shard.yml

```
@@ -4,6 +4,6 @@ version: 0.1.0
 authors:
   - Brian J. Cardiff <bcardiff@manas.tech>
 
-crystal: 0.34.0
+crystal: ">= 0.34.0"
```

```suggestion
crystal: ">= 0.34.0, < 2.0.0"
```

I would rather keep the upper bound.

Then we can release 1.0.1.

---

Sure, I've just updated the MR. FYI, I made the same change in https://github.com/crystal-lang/yaml_mapping.cr/pull/2.